tests/etl/test_perf_data_adapters.py

```diff
@@ -50,4 +51,22 @@ def test_adapt_and_load():
         datum['blob'] = json.dumps({'talos_data': [datum['blob']]})
         tda.adapt_and_load(reference_data, job_data, datum)
 
+        # we upload a summary with a suite and subtest values, +1 for suite
+        if 'summary' in datum['blob']:
```

Comment:
I think this condition will always be true with your patch? Ideally we want to test both codepaths: one where the talos blob has a summary, one where it doesn't/